src/Devices/SD_TC.h

```diff
@@ -1,7 +1,9 @@
 #pragma once
 
 #include <Arduino.h>
-#include <SD.h>
+
+#define SS 4
```

Comment:
Why do we need the define statement?

Reply:
Somehow it wasn't needed for (or was defined by) the SD library (which is GPL). When we switched to using the SdFat library (LGPL) there was a compile error that `SS` was not defined. I suspect that it is something that is defined by one of the real Arduino header files but is not defined by the Arduino_CI header files. When looking at its sole use, it is the default for a constructor (or begin) that specifies a pin. I looked up what we were using and assigned it. That was as far as I got.